seo-app/image_scrapping.py

```diff
@@ -0,0 +1,52 @@
+from attr import attr
+from bs4 import BeautifulSoup 
+import requests
+
+
+class ScrappedImage:
+    def __init__(self, url, img_source=None, alt_text=None):
+       self.url = url
+       self.img_source = img_source
+       self.alt_text = alt_text
+
+    def get_image_attributes(self):
+       reqs = requests.get(self.url)
+       soup = BeautifulSoup(reqs.text, 'html.parser')
+       
+       image_tag = soup.find('div', class_="banner-img").find('img')          
```

Comment:
Not every URL's image tag will be enclosed in class named `banner-image`. Anyway, this attribute is already passed to the class on initialization.